src/Cashier.php

```diff
@@ -3,10 +3,25 @@
 namespace Laravel\Cashier;
 
 use Exception;
+use Illuminate\Support\Str;
+use Laravel\Cashier\Exception as CashierException;
+use Laravel\Cashier\Gateway\Gateway;
 
+/**
+ * Class Cashier
+ *
+ * @package Laravel\Cashier
+ */
 class Cashier
```

Comment:
I'm not a huge fan of having an all-static `Cashier` object. I'll explore options—maybe a facade.